adaptive.css

```diff
@@ -0,0 +1,89 @@
+@media (max-width: 375px) {
+  .logo__circle logo__circle_color_dark-blue,
+  .logo__circle logo__circle_color_blue {
+    margin-top: 25px;
+  }
+
+  .main {
+    padding: 34% 0 0;
+    background-size: 34%;
+    background-position: top center;
+  }
+
+  .main__title {
+    font-size: 18px;
+    line-height: 25px;
+    margin-bottom: 20;
+  }
+
+  .main__subtitle {
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .main__button {
+    margin: -7px auto 0 -12px;
+  }
+}
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .header {
+    margin-bottom: 28px;
+  }
+
+  .logo__shadow {
+  display: none;
+  }
+
+  .burger {
+    display: block;
+    width: 25px;
+    height: 17px;
+    padding-top: 33px;
+  }
+
+  .main {
+    padding: 74% 0 0;
+    background-size: 74%;
+    background-position: top center;
+  }
+
+  .main__title {
+    margin-bottom: 44px;
+    font-family: PT Serif, Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-size: 30px;
+    line-height: 40px;
+    text-align: center;
+  }
+
+  .main__subtitle {
+    margin-bottom: 25px;
+    font-size: 16px;
+    line-height: 19px;
+    text-align: center;
+  }
+  
+  .main__button {
+    position: absolute;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16px;
+    margin: 10px auto 0 122px;
+  }
+}
+
+@media (max-width: 1170px) {
+  .main {
+    padding: 45% 0 0;
+    background-size: 45% auto;
+    background-position: right;
+  }
+
+  .main__title {
+
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6240796)